metric_config_parser/tests/integration/test_config_integration.py

```diff
@@ -4,16 +4,9 @@
 class TestConfigIntegration:
     def test_configs_from_repo(self):
         config_collection = ConfigCollection.from_github_repos(
-            ["https://github.com/mozilla/metric-hub", "https://github.com/mozilla/jetstream-config"]
+            ["https://github.com/mozilla/metric-hub"]
         )
```

Comment:
I just added support for pulling configs from a repo sub-directory by passing something like `https://github.com/mozilla/metric-hub/tree/main/jetstream`. So I think we'll want to keep this as it will be used in jetstream to merge metric-hub and jetstream configs.